content/en/real_user_monitoring/guide/monitor-your-nextjs-app-with-rum.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Monitor Your Next.js App With RUM
+kind: guide
+description: Guide for monitoring Next.js applications with RUM.
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM Monitors'
+---
+
+## Overview
+[Next.js][1] is a JavaScript framework that is used to create React.js web pages and Node.js APIs. You can integrate Next.js with RUM to monitor your frontend applications for browser-related metrics that you give insight into performance and user behavior.
+
+## Setup
+
+To set up Datadog RUM browser monitoring:
+
+1. Navigate to **[UX Monitoring > Real User Monitoring][2]**.
+2. Click the **New Application** button.
+3. Make sure JS is selected, then enter a name for your application and click **Create New RUM Application**. A `clientToken` and `applicationId` are automatically generated for your application.
+4. Instrument your application through [npm][3], [CDN async][4], or [CDN sync][5].
```

Comment:
These steps are a little bit different for Next.js so I think we want to make it more clear what the customer needs to do for each. 

If using NPM, the customer can copy / paste the code snippet from the DD RUM UI into either
a) The root layout.tsx file (https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required) if using the newer Next.js app router https://nextjs.org/docs/app
b) A custom _app.tsx file (https://nextjs.org/docs/pages/building-your-application/routing/custom-app#usage) if using the older Next.js page router https://nextjs.org/docs/pages

If using CDN sync or async, the customer needs to make a few small changes to the code snippet and then paste them into either layout.tsx if using the app router or _app.tsx if using the older page router. The difference from the code snippet in the UI is that Next.js external scripts need to be loaded like this: https://nextjs.org/docs/messages/next-script-for-ga#using-analyticsjs-legacy. We could maybe write something like this in the doc:
```tsx
<Script id="datadog-rum">
    {`
          // Paste CDN sync or async snippet here without first <script> and last </script> lines
    `}
</Script>
```